app/pages/confirm.jsx

```diff
@@ -36,38 +81,53 @@ class Confirm extends React.Component {
             travel: !needsReimbursement
         };
 
-        return (
-            <PageContainer>
-                <FormContainer>
-                    {isConfirmed
-                        ? <Alert message="You are confirmed!" positive={true} />
-                        : null}
-                    <h2>Confirm Attendance at MHacks X!</h2>
-                    <MHForm
-                        schema={ConfirmAttendanceSchema}
-                        hidden={hiddenFields}
-                        theme={this.props.theme}
-                        onSubmit={this.onSubmit}
-                    />
-                    <p>
-                        By confirming attendance, you also confirm you have read
-                        and agree to the{' '}
-                        <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">
-                            MLH Code of Conduct
-                        </a>, the{' '}
-                        <a href="https://github.com/MLH/mlh-policies/blob/master/data-sharing.md">
-                            MLH Data Sharing Provision
-                        </a>, the{' '}
-                        <a href="https://docs.google.com/document/d/1L9wC7lfXmOBCKdUQancuoYQf86KIQqUJ0is4dr8QqQM/pub">
-                            MHacks Code of Conduct
-                        </a>, and the{' '}
-                        <a href="https://docs.google.com/document/d/1tFmaC_eVVPJ9fKnjGeYHRX2_th3-JOd9ZP-IoTEM91I/pub">
-                            MHacks Liability Waiver
-                        </a>.
-                    </p>
-                </FormContainer>
-            </PageContainer>
-        );
+        return !this.props.userState.data.form &&
```

Comment:
nit: can you change this to a return early rather than a ternary? It's just a style thing but I feel like early exits should be separated